packages/collector/src/agentConnection.js

```diff
@@ -266,6 +266,8 @@ exports.sendSpans = function sendSpans(spans, cb) {
   const callback = util.atMostOnce('callback for sendSpans', err => {
     if (err && !maxContentErrorHasBeenLogged && err instanceof PayloadTooLargeError) {
       logLargeSpans(spans);
+    } else if (err && !maxContentErrorHasBeenLogged) {
```

Comment:
We require span logs even when there are no errors. Sometimes, tracing does not appear in the UI, so we need to verify if the spans were sent. It would be beneficial to add debug logs for this purpose.

Reply:
If we add span logs for all cases, then it could be a bit difficult to find the error cases.
One solution is logging error cases in `warn`/`error` and logging everything else under `debug`.
how does that sound? any other suggestions?

Reply:
updated code